network/src/proto/mempool.proto

```diff
@@ -3,7 +3,7 @@
 
 syntax = "proto3";
 
-package network;
+package mempool;
```

Comment:
Was this a bug you fixed? I don't quite understand why the package name would change, although it seems like it should be mempool not network.

Reply:
I honestly don't know what the package thing is for. It seems like rust-protobuf ignores it and prost expects the generated code to be in a module named after the package used in the .proto file